packages/web-app-epub-reader/src/App.vue

```diff
@@ -4,36 +4,76 @@
       <li
         v-for="chapter in chapters"
         :key="chapter.id"
-        class="epub-reader-chapters-list-item oc-mt-xs"
+        class="epub-reader-chapters-list-item oc-py-s"
         :class="{ active: currentChapter.id === chapter.id }"
       >
         <oc-button class="oc-text-truncate" appearance="raw" @click="showChapter(chapter)">
           <span
             v-oc-tooltip="chapter.label"
-            class="oc-text-truncate oc-text-small oc-mr-s"
+            class="oc-text-truncate oc-mr-s"
             v-text="chapter.label"
           />
         </oc-button>
       </li>
     </oc-list>
-    <div class="oc-flex oc-flex-center oc-width-1-1">
-      <div class="oc-flex oc-flex-middle oc-mx-l">
-        <oc-button :disabled="navigateLeftDisabled" appearance="raw" @click="navigateLeft">
-          <oc-icon name="arrow-left-s" fill-type="line" size="xlarge" />
-        </oc-button>
+    <div class="oc-width-1-1 oc-height-1-1">
+      <div class="epub-reader-controls oc-flex oc-flex-middle oc-m-s">
+        <div class="oc-flex oc-button-group">
+          <oc-button
+            v-oc-tooltip="`${fontSizePercentage - FONT_SIZE_PERCENTAGE_STEP}%`"
+            class="epub-reader-controls-font-size-decrease"
+            :disabled="decreaseFontSizeDisabled"
+            gap-size="none"
+            @click="decreaseFontSize"
+          >
+            <oc-icon name="font-family" fill-type="none" size="small" />
+            <oc-icon name="subtract" size="xsmall" />
+          </oc-button>
+          <oc-button
+            v-oc-tooltip="$gettext('Reset font size')"
+            class="epub-reader-controls-font-size-reset"
+            gap-size="none"
+            @click="resetFontSize"
+            v-text="`${fontSizePercentage}%`"
+          />
+          <oc-button
+            v-oc-tooltip="`${fontSizePercentage + FONT_SIZE_PERCENTAGE_STEP}%`"
+            class="epub-reader-controls-font-size-increare"
+            :disabled="increaseFontSizeDisabled"
+            gap-size="none"
+            @click="increaseFontSize"
+          >
+            <oc-icon name="font-family" fill-type="none" size="small" />
+            <oc-icon name="add" size="xsmall" />
+          </oc-button>
+        </div>
+        <oc-select
+          v-model="currentChapter"
+          class="epub-reader-controls-chapters-select oc-width-1-1 oc-px-s oc-hidden@l"
```

Comment:
Don't know why, but I don't have a `pointer` cursor when hovering over the OcSelect. Could you add the `oc-cursor-pointer` class and check if that helps?

Reply:
Could you compare to our other oc selects ? I don't see a difference (even if cursor is not a pointer but smth else)

Reply:
Aaaah, found it. The `:searchable="false"` attribute is missing. :-) Please add it, no free text input allowed for this dropdown, only selecting the predefined options is allowed.